src/controllers/indexController.js

```diff
@@ -11,13 +11,18 @@ import {
   _getPlayground,
   _getRestart,
 } from "@/controllers/actionController";
+import { getRandomStat } from "@/utils/randomStat.util";
 
 export const home = async (req, res, next) => {
   try {
     const isAuthenticated = Boolean(req.player);
     return res.render("home", {
       gameTitle: "여기에 게임 이름 입력",
       isAuthenticated,
+      hp: req.player ? req.player.hp : null,
+      str: req.player ? req.player.str : null,
+      def: req.player ? req.player.def : null,
+      randomStatNum: req.player ? 6 - req.player.randomStatNum : null,
     });
```

Comment:
아래와 같은 꼴이 더 가독성이 좋을 것 같아요
```suggestion
    if (req.player) {
       return res.render(...)
    }
    return res.render(...) 
```